history-api/script.js

```diff
@@ -0,0 +1,43 @@
+// Update the page with the new content
+function displayContent(content) {
+  const description = document.querySelector("#description");
```

Comment:
Let's change the title too. That way, the history will be more useful:
![Capture d’écran 2023-11-17 à 08 23 41](https://github.com/mdn/dom-examples/assets/1466293/79f01379-3289-4950-b1a0-c9b594807f77)


```suggestion
  document.title = `Creatures: ${content.name}`;
  
  const description = document.querySelector("#description");
```